android/app/build.gradle

```diff
@@ -43,46 +47,50 @@ android {
-
-  implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-
-  implementation "android.arch.lifecycle:runtime:1.1.1"
-  implementation "android.arch.lifecycle:extensions:1.1.1"
-  kapt "android.arch.lifecycle:compiler:1.1.1"
-
-  implementation "org.jetbrains.anko:anko-sdk25:$anko_version"
-  implementation "org.jetbrains.anko:anko-appcompat-v7:$anko_version"
-  implementation "org.jetbrains.anko:anko-recyclerview-v7:$anko_version"
-  implementation "org.jetbrains.anko:anko-commons:$anko_version"
-  implementation "org.jetbrains.anko:anko-design:$anko_version"
-  implementation "org.jetbrains.anko:anko-coroutines:$anko_version"
-
-  implementation 'com.squareup.retrofit2:retrofit:2.4.0'
-  implementation 'com.squareup.retrofit2:converter-gson:2.4.0'
-  implementation 'com.squareup.okhttp3:logging-interceptor:3.10.0'
-
-  implementation 'net.opacapp:multiline-collapsingtoolbar:1.5.0'
-  implementation 'com.github.bumptech.glide:glide:3.7.0'
-  implementation 'com.brandongogetap:stickyheaders:0.4.9'
-  implementation 'ru.gildor.coroutines:kotlin-coroutines-retrofit:0.11.0'
-    compile "org.jetbrains.kotlinx:kotlinx-coroutines-android:$coroutines_version"
-
-  implementation project(":common-jvm")
-}
+    expectedBy project(':common')
+
+    compile 'com.android.support:multidex:1.0.3'
+
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+
+    implementation 'com.android.support:appcompat-v7:28.0.0-rc01'
+    implementation 'com.android.support:support-v4:28.0.0-rc01'
+    implementation 'com.android.support:recyclerview-v7:28.0.0-rc01'
+    implementation 'com.android.support.constraint:constraint-layout:1.1.2'
+
+    kapt 'android.arch.lifecycle:compiler:1.1.1'
+
+    implementation 'com.android.support:design:28.0.0-rc01'
 
-kotlin {
-  experimental {
-    coroutines "enable"
-  }
-}
\ No newline at end of file
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+
+    implementation 'android.arch.lifecycle:runtime:1.1.1'
+    implementation 'android.arch.lifecycle:extensions:1.1.1'
+
+    implementation "org.jetbrains.anko:anko-sdk25:$anko_version"
+    implementation "org.jetbrains.anko:anko-appcompat-v7:$anko_version"
+    implementation "org.jetbrains.anko:anko-recyclerview-v7:$anko_version"
+    implementation "org.jetbrains.anko:anko-commons:$anko_version"
+    implementation "org.jetbrains.anko:anko-design:$anko_version"
+    implementation "org.jetbrains.anko:anko-coroutines:$anko_version"
+
+    implementation 'net.opacapp:multiline-collapsingtoolbar:1.5.0'
+    implementation 'com.github.bumptech.glide:glide:3.7.0'
+    implementation 'com.brandongogetap:stickyheaders:0.4.9'
+
+
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$coroutines_version"
+    implementation "io.ktor:ktor-client-android:$ktor_version"
+    implementation "io.ktor:ktor-client-gson:$ktor_version"
+
+    api "io.ktor:ktor-client-json-jvm:$ktor_version"
```

Comment:
You have both `io.ktor:ktor-client-gson` and `io.ktor:ktor-client-json-jvm`, is this intended?